src/main/java/racingcar/model/PlayTime.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.model;
+
+public class PlayTime {
+
+    private static final int MIN_PLAY_TIME = 1;
+    private static final String MIN_PLAY_TIME_ERROR_MESSAGE = "[ERROR] 레이싱 횟수는 1회 이상이어야 합니다.";
+
+    private Integer playTime;
+
+    public PlayTime(Integer playTime) {
+        if (playTime < MIN_PLAY_TIME) {
+            throw new IllegalArgumentException(MIN_PLAY_TIME_ERROR_MESSAGE);
+        }
+        this.playTime = playTime;
+    }
+
+    public boolean isZero() {
+        return playTime == 0;
+    }
+
+    public void decreasePlayTime() {
```

Comment:
뒤에 PlayTime을 붙이지 않아도 값 객체라서 의도가 잘 드러날 것 같아요!
```suggestion
    public void decrease() {
```

Reply:
값 객체에 대해 많이 서툴러서 해당 메소드명을 사용하게 되었습니다. 지적 감사드립니다!!